code/modules/mapping/mapping_helpers.dm

```diff
@@ -732,3 +731,12 @@ INITIALIZE_IMMEDIATE(/obj/effect/mapping_helpers/no_lava)
 	var/turf/open/floor/floor = get_turf(src)
 	floor.burn_tile()
 	qdel(src)
+
+// Used by mapmerge2 to denote the existence of a merge conflict (or when it has to complete a "best intent" merge where it dumps the movable contents of an old key and a new key on the same tile).
+// We define it explicitly here to ensure that it shows up on the highest possible plane (while giving off a verbose icon) to aide mappers in resolving these conflicts.
+// DO NOT USE THIS IN NORMAL MAPPING!!! Linters WILL fail.
+/obj/effect/mapping_helpers/merge_conflict_marker
+	name = "Merge Conflict Marker - DO NOT USE"
+	icon_state = "merge_conflict_marker"
+	desc = "If you are seeing this in-game: someone REALLY, REALLY, REALLY fucked up. They physically mapped in a fucking Merge Conflict Marker. What the shit."
```

Comment:
why is this a mapping helper? someone will not be able to see this and report an error because mapping helpers immediately delete themselves

Reply:
i might have an idea for this then